samples/drivers/video/sw_pipeline/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(usb_video)
+
+include(${ZEPHYR_BASE}/samples/subsys/usb/common/common.cmake)
+target_sources(app PRIVATE src/main.c)
+target_sources(app PRIVATE src/pipeline.c)
```

Comment:
```suggestion
target_sources(app PRIVATE src/main.c src/pipeline.c)
```

Reply:
Good to know about this, I will preserve this style in future PRs.